src/main/java/io/nats/client/support/JsonUtils.java

```diff
@@ -540,37 +540,41 @@ private static String indent(int level) {
         return level == 0 ? "" : INDENT.substring(0, level * 4);
     }
 
-    public static void printFormatted(Object o) {
+    public static String getFormatted(Object o) {
```

Comment:
not kv related. This util was refactored so formatted json could be used in context other than just printing it. Should have done this originally.